### ISSUE TITLE
Pass arguments to the tests

### ISSUE THREAD

**Issue by thofma** [oscar-system/OscarDevTools.jl#1]
julia >= 1.3 can pass arguments to the test process by specifying the `test_args::Union{Cmd, Vector{String}}` keyword arguments for `Pkg.test`.

It would be great if the configuration toml would have a way to specify the arguments for the tests of an individual package. E.g.

```toml
[testoptions]
Hecke = ["short"]
```

The reason is that the Hecke tests are way too long to be useful for the upstream package testing. So I would like to limit them.

I can have a stab at it if we agree that this is a useful feature.

**Comment by benlorenz** [comment]
I like this and will implement that with a slightly different configuration syntax:, e.g.:
```toml
[pkgs]
  [pkgs.Oscar]
  branches = [ "master", "release" ]
  test = true
  
  [pkgs.Hecke]
  branches = [ "master", "release" ]
  test = true
  testoptions = [ "short" ]
```

This also allows specifying which packages should be tested, which is currently hardcoded to just Oscar.

**Comment by thofma** [comment]
Nice. I like it. Now I only need to implement the test option handling in Hecke.

**Comment by benlorenz** [comment]
Test options should be working in the [`dev`](https://github.com/oscar-system/OscarDevTools.jl/tree/dev) branch now, see here for an example: https://github.com/oscar-system/Polymake.jl/pull/343/files

I think it can be used in `runtests.jl` by doing something like `"short" in ARGS`.
(The `test_args` flag produces the following extra code when running the tests: `append!(empty!(ARGS), ["short"])`)

I will try to add some kind of testcases to the repo here and an initial readme soon.

**Comment by thofma** [comment]
Thanks, that looks great. I have the `ARGS` thing with `"short"` working in Hecke, just need a new release.

**Comment by benlorenz** [comment]
fixed via #4
